React Native/Custom Fonts.md

```diff
@@ -0,0 +1,37 @@
+## Custom fonts
+
+For this tutorial we will be using custom font that can be 
+downloaded here https://www.onlinewebfonts.com/download/3a88649e176a40a6d80b395ca0ae430d
+
+### Setup
+- To make font crossplatform - you need to have a proper name for it.
+Between font name and type you should have a dash. For example:  
+`DINNextLTProBold.ttf` - wrong  
+`DINNextLTPro-Bold.ttf` - correct
+
+- Add font file in “assets/fonts” directory
+
+- Define assets directory. Create File `react-native.config.js` and add following code:
+
+```
+module.exports = {
+  project: {
+    ios: {},
+    android: {}, // grouped into "project"
+  },
+  assets: ["./assets/fonts/"], // stays the same
+};
+```
+
+- Link assets using react native link. Run in terminal:
+
+`react-native link`
+
+- Now you can use your font in your styles:
```

Comment:
I would also explicitly say that when using custom fonts, css rule `font-weight` doesn't apply. To use different font `weight` we need to add separate font file with that weight, and then change the `font-familiy` in css to apply.